docs/pages/server-access/guides/openssh/openssh-manual-install.mdx

```diff
@@ -18,6 +18,27 @@ and running, but in the long run, we would recommend replacing `sshd` with `tele
 - [Advanced session recording](bpf-session-recording.mdx)
 - [Restricting outbound network connections in SSH sessions](restricted-session.mdx)
 
+Teleport supports OpenSSH by proxying SSH connections through the Proxy Service. When a Teleport user requests to connect to an OpenSSH node, the Proxy Service checks the user's Teleport roles.
+
+If the RBAC checks succeed, the Proxy Service authenticates to the OpenSSH node with a dynamically generated certificate signed by a Teleport CA. This allows the
+Proxy Service to record and audit connections to OpenSSH nodes.
+
+The Proxy Service prevents Teleport users from bypassing auditing by requiring
+a certificate signed by a Teleport CA that only the Auth Service possesses.
+
+In this setup, the Teleport SSH Service performs RBAC checks as well as audits and records sessions on its host, which eliminates the need for connection termination when recording SSH sessions.
+
+<Admonition
+  type="tip"
+  title="Note"
+>
+
+  This guide shows you how to register an OpenSSH node manually, but if you can
+  copy the `teleport` binary onto your OpenSSH node and execute it following the
+  [standard registration guide](openssh.mdx) will be much easier.
```

Comment:
I would make this more precise:
- What does "manually" mean in this case? Maybe say, "...by configuring OpenSSH to trust the Teleport CA"?
- I would use another wording than "will be much easier", since we often can't predict what will trip up users, all aspects of their environment, etc. Maybe say, "will not require you to maintain SSH credentials" or something else that describes the situation more concretely?

Reply:
Done, look good now?